telegrambots/src/main/java/org/telegram/telegrambots/bots/DefaultAbsSender.java

```diff
@@ -123,7 +120,11 @@ public final DefaultBotOptions getOptions() {
     }
 
     public String getBaseUrl() {
-        return options.getBaseUrl() + getBotToken() + "/";
+        String baseUrl = options.getBaseUrl() + getBotToken() + "/";
```

Comment:
Does not it better to allow user set url for http request on start? And don't add implicit changes. What if in future telegram decide to change url again.